### PR TITLE
add Website Updates WG

### PR DESCRIPTION
**Pull request by durandom**
cc @hpdempsey @quaid @stefwrite @bryanmontalvan

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -23,7 +23,19 @@ sigs:
...
+  subprojects:
+  - name: website
+    description: |-
+      Enhance the user experience of the operate-first.cloud website.
+    owners:
+    - https://raw.githubusercontent.com/operate-first/operate-first.github.io/master/OWNERS
+    meetings:
+    - description: website meeting
+      day: Tuesday
+      time: "8:30"
```
Comment (durandom) [inline sigs.yaml:35]: @hpdempsey can you find a timeslot for us

File: sigs.yaml [hunk 2]
```diff
@@ -23,7 +23,19 @@ sigs:
...
+      tz: EST
+      frequency: Weekly
+      url: meet.google.com/....
```
Comment (durandom) [inline sigs.yaml:38]: @quaid can you setup a standing meeting URL?

**Comment by durandom** [comment]
@quaid @hpdempsey bump on review and comments

**Comment by durandom** [comment]
@hpdempsey @quaid @stefwrite @bryanmontalvan  please review again

**Review by schwesig — APPROVED** [review]
/lgtm
/approve

**Pull request merged**